.github/workflows/tests+artifacts+pypi.yml

```diff
@@ -263,26 +263,26 @@ jobs:
       - if: startsWith(matrix.platform, 'ubuntu-')
         run: echo NUMBA_THREADING_LAYER=omp >> $GITHUB_ENV
 
-      - if: startsWith(matrix.platform, 'ubuntu-') && matrix.test-suite == 'multi-process_a'
+      - if: startsWith(matrix.platform, 'ubuntu-') && (matrix.test-suite == 'multi-process_a' || matrix.test-suite == 'multi-process_c')
```

Comment:
Paraview is only used in `Arabas_et_al_2015` and `Arabas_et_al_2023` examples, no need to add it here

Reply:
`Multi-process_a` was divided into `a` and `c`, so I've added it to all if statements. But will delete this change if it's not needed.